### PR TITLE
issue-105: Adding a default page for authorization errors

### PR DESCRIPTION
**Pull request by austinportal**

### CODE REVIEW

File: servlet/application.properties [hunk 1]
```diff
@@ -202,6 +202,8 @@ suit.attivio.users=
 # the URL might be "https://.myhost.com/searchui".
 #saml.sso.metadata-generator.entity-base-url=
 
+# The page to navigate to if an unhandled authentication error
+#saml.sso.defaultFailureUrl=/autherror.html
```
Comment (luigivaldez) [inline servlet/application.properties]: I don't see any reason to not have this uncommented by default... (e.g., active).

File: frontend/src/autherror.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (luigivaldez) [inline frontend/src/autherror.html]: We might want to make this a teeny bit spiffier… I think the basic fact that there was an error should be in the page content and not just in the title of the tab/window.

Maybe something like:

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Error</title>
  </head>
  <body style="margin: 0; height: 100vh; width: 100vw; text-align: center;">
    <div style="height: 30vh;">
      &nbsp;
    </div>
    <div style="margin-left: auto; margin-right: auto">
      <h1>An Error Occurred</h1>
      <h3>Elevated permissions are required to see this content.</h3>
      <p>Please contact your system administrator for more information.</p>
    </div>
  </body>
</html>
```

File: servlet/application.properties [hunk 3]
```diff
@@ -95,7 +95,7 @@ suit.attivio.password=attivio
 # or the main HTML file. Generally, the list of routes here should match the
 # routes defined in your application's main component (e.g. SearchUIApp.js in the
 # out-of-the-box version). It is a comma-separated list (with no spaces).
-suit.attivio.routes=/,/landing,/results,/results-no-mast,/insights,/doc360,/login,/error
+suit.attivio.routes=/,/landing,/results,/results-no-mast,/insights,/doc360,/login
```
Comment (luigivaldez) [inline servlet/application.properties]: We should make a corresponding change to remove `/error` from the default values used when this isn't set in the properties file. See the `DEFAULT_ROUTES` constant in `HomeControllerHandlerMapper.java`...

**Comment by austinportal** [comment]
Looking into if it's possible to differentiate the type authentication exceptions

**Pull request merged**